### PR TITLE
Update Basic Sample to use WebApplicationBuilder

### PR DESCRIPTION
**Pull request by hananiel**
Updates the basic sample to use the WebApplicationBuilder.

### CODE REVIEW

File: Stream/Basic/Readme.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+﻿# Basic Stream Sample using WebApplication Builder
+
+This sample assumes you have rabbitMQ broker running with default credentials.
```
Comment (TimHess) [inline Stream/Basic/Readme.md:3]:
```suggestion
This sample assumes you have RabbitMQ broker running with default credentials. 
```

File: Stream/Basic/MyStreamProcessor.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+﻿using Castle.Core.Logging;
```
Comment (TimHess) [inline Stream/Basic/MyStreamProcessor.cs:1]: I'm not familiar with this one, is this using intentional?

File: Stream/Basic/MyStreamProcessor.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        {
+            _logger = logger;
+        }
+
+        [StreamListener(IProcessor.INPUT)]
+        [SendTo(IProcessor.OUTPUT)]
+        public string Handle(string input)
+        {
+            var output = input.ToUpper();
+            _logger?.LogInformation($"MyStreamProcssor changed input:{input} into output:{output} ");
```
Comment (TimHess) [inline Stream/Basic/MyStreamProcessor.cs:31]:
```suggestion
            _logger?.LogInformation($"MyStreamProcessor changed input:{input} into output:{output} ");
```

File: Stream/Basic/MyStreamProcessor.cs (hunk 2 above)
Comment (hananiel) [reply Stream/Basic/MyStreamProcessor.cs]: fixed. thanks!

**Review by TimHess — APPROVED** [review]

**Pull request merged**